src/components/tooltip/_tooltip.scss

```diff
@@ -117,7 +122,7 @@
       top: 50%;
       left: $tooltip-margin;
       transform: translate(0, -50%);
-      &:after {
+      &:before {
```

Comment:
why does replacing after with before work ?

Reply:
because i want to add another css under the after, i added inset around the tooltip so user can move inside the tooltip before it disappear.
It does not affect the old behavior.